.github/workflows/tests.yml

```diff
@@ -27,30 +24,15 @@ jobs:
 
     - name: Set up Homebrew
       run: |
+        cd $(brew --repo)
+        git fetch --tags origin ${{github.sha}}
+        git checkout --force FETCH_HEAD
+        git branch -f master HEAD
+        brew update-reset Library/Taps/homebrew/homebrew-core
+        git log -1
         if [ "$RUNNER_OS" = "Linux" ]; then
-          HOMEBREW_REPOSITORY=/home/linuxbrew/.linuxbrew/Homebrew
-
-          sudo mkdir -p "$HOMEBREW_REPOSITORY/Library/Taps"
-          mv "$HOMEBREW_REPOSITORY/Library/Taps" "$PWD/Library"
-          sudo rm -rf "$HOMEBREW_REPOSITORY"
-          sudo ln -sf "$PWD" "$HOMEBREW_REPOSITORY"
-
-          cd /home/linuxbrew/.linuxbrew
-          sudo mkdir -p bin etc include lib opt sbin share var/homebrew/linked Cellar
-          sudo ln -sf ../Homebrew/bin/brew /home/linuxbrew/.linuxbrew/bin/
-          sudo chown -R "$USER" /home/linuxbrew
-
-          brew update-reset Homebrew/Library/Taps/homebrew/homebrew-core
-        else
-          HOMEBREW_REPOSITORY="$(brew --repo)"
-
-          mv "$HOMEBREW_REPOSITORY/Library/Taps" "$PWD/Library"
-          sudo rm -rf "$HOMEBREW_REPOSITORY"
-          sudo ln -sf "$PWD" "$HOMEBREW_REPOSITORY"
-
-          brew update-reset Library/Taps/homebrew/homebrew-core
+          sudo chown -R "$USER" $(brew --prefix)/*
```

Comment:
Could be just `$(brew --prefix)` on Linux or `$(brew --prefix)/*` on both macOS and Linux.

Reply:
Running `chmod` on `brew --prefix` on macOS runners takes about a minute (compared to nearly instantly on Linux), because I think a lot of other stuff is installed here by default.

Reply:
Cool. Can bin it, then and just do the `sudo chown -R "$USER" $(brew --prefix)` on Linux.